### PR TITLE
Minor fixes and style changes

### PR DESCRIPTION
**Pull request by fekete-robert**

### CODE REVIEW

**Comment by fekete-robert** [comment]
@akijakya: a PR against your add-docs PR with minor changes, I just could't add you as a reviewer directly

File: README.md [hunk 1]
```diff
@@ -1,19 +1,18 @@
 # Vault Secrets Reloader
 
 Vault Secrets Reloader can periodically check if a secret that is used in watched workloads has a new version in
-Hashicorp Vault, and if so, automatically “reloads” them by incrementing an annotation value, initiating a rollout for
-the workload’s pods. This controller is essentially a complementary to `vault-secrets-webhook`, relying on it for
+Hashicorp Vault, and if so, automatically “reloads” them by incrementing an annotation value, then initiating a rollout for
```
Comment (akijakya) [inline README.md:4]: I think the original is more correct here, as incrementing the annotation value is the act that triggers Kubernetes for a new rollout, not a following step done by the Reloader.

File: README.md [hunk 2]
```diff
@@ -1,19 +1,18 @@
...
-If you already use the [webhook](https://github.com/bank-vaults/vault-secrets-webhook), you are probably aware of it
-only injecting secrets when the pods are created/recreated, and until now, there were no solution within the Bank-Vaults
+If you already use [`vault-secrets-webhook`](https://github.com/bank-vaults/vault-secrets-webhook), you probably know that it
+only injects secrets when the pods are created/recreated. Until now, there was no solution within the Bank-Vaults
 ecosystem to inject secrets into these workloads in a continuous manner. Vault Secrets Reloader offers Vault Secrets
 Webhook users an automated solution for this problem.
 
-> [!IMPORTANT] This is an **early alpha version** and breaking changes are expected. As such, it is not recommended
-> for usage in production.
+> **IMPORTANT!** This is an **early alpha version** and breaking changes are expected. As such, do not use it in production.
```
Comment (akijakya) [inline README.md:13]: I tried to use a Github markdown feature here (https://github.com/orgs/community/discussions/16925) but Ramiz already corrected it on the main PR 🙂

**Review by akijakya — CHANGES_REQUESTED** [review]
Thanks for the corrections!

File: README.md (hunk 2 above)
Comment (fekete-robert) [reply README.md:13]: I didn't realize that, we can leave the original in that case, and will update it if needed when we move the content to the hugo site.

**Review by akijakya — APPROVED** [review]
Thanks!

**Pull request merged**